test/writeIndex.js

```diff
@@ -8,29 +8,59 @@ import {
 import writeIndex from '../src/utilities/writeIndex';
 import codeExample from './codeExample';
 
+const readFileByPath = (filePath) => {
+  return fs.readFileSync(filePath, 'utf8');
+};
+
+const removeFileByPath = (filePath) => {
+  // eslint-disable-next-line no-empty
+  try {
```

Comment:
I was trying to follow "code style" in the project so if you look here https://github.com/gajus/create-index/pull/37/files#diff-f546cc5537a07f2741198b32e7ef3aa9L17 the same pattern with swallowing an error has been already used. I decided to just copy and use it.

Anyway I will remove them if you want, no problem :)